mwdb/web/src/components/ShowObject/Views/RelatedFilesTab.js

```diff
@@ -0,0 +1,204 @@
+import React, { useCallback, useContext, useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+    faPlus,
+    faExternalLinkSquare,
+    faDownload,
+    faTrash,
+} from "@fortawesome/free-solid-svg-icons";
+
+import { ObjectContext } from "@mwdb-web/commons/context";
+import { ObjectAction, ObjectTab } from "@mwdb-web/commons/ui";
+import { APIContext } from "@mwdb-web/commons/api/context";
+import { humanFileSize } from "@mwdb-web/commons/helpers";
+import ReactModal from "react-modal";
+
+async function updateRelatedFiles(api, context) {
+    const { setObjectError, updateObjectData } = context;
+    try {
+        let response = await api.getListOfRelatedFiles(context.object.sha256);
+        updateObjectData({
+            related_files: response.data.related_files,
+        });
+    } catch (error) {
+        setObjectError(error);
+    }
+}
+
+function ShowRelatedFiles() {
+    const api = useContext(APIContext);
+    const context = useContext(ObjectContext);
+    const { setObjectError, updateObjectData } = context;
+
+    function RelatedFileItem({ file_name, file_size, sha256 }) {
+        return (
+            <tr>
+                <td>{file_name}</td>
+                <td>{humanFileSize(file_size)}</td>
+                <td>
+                    <Link
+                        to={`/file/${context.object.sha256}/related_files`}
+                        className="nav-link"
+                        onClick={async () => {
+                            let content = await api.downloadRelatedFile(sha256);
+                            let blob = new Blob([content.data], {
+                                type: "application/octet-stream",
+                            });
+                            let tempLink = document.createElement("a");
+                            tempLink.style.display = "none";
+                            tempLink.href = window.URL.createObjectURL(blob);
+                            tempLink.download = file_name;
+                            tempLink.click();
```

Comment:
Code duplication: I would use download helper (or modify that helper if we cannot use it directly)

Reply:
Done